### PR TITLE
Bug 1198164 - Include MetricsLevel in FtuPing. r=marshall.

### PR DESCRIPTION
**Pull request by tamarahills**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=703006825dd5ca468263c00f41ef85909d58e94b

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=27bd1a42243af3f4508ea37750ac41a84f11e394

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=253bb28a065041a698e5cea29302023cfdc4ad4f

File: apps/system/js/ftu_ping.js [hunk 1]
```diff
@@ -132,7 +133,8 @@
       function getSettings() {
         return new Promise(function(resolve, reject) {
           var allSettings = [FTU_PING_URL, FTU_PING_TRY_INTERVAL,
-                             FTU_PING_TIMEOUT, FTU_PING_MAX_NETWORK_FAILS].
+                             FTU_PING_TIMEOUT, FTU_PING_MAX_NETWORK_FAILS,
+                             FTU_PING_TELEMETRY_ENABLED_KEY].
```
Comment (russnicoletti) [inline apps/system/js/ftu_ping.js]: Nit: code would be more readable with each of the variables being on a separate line.

File: apps/system/js/ftu_ping.js [hunk 2]
```diff
@@ -19,6 +19,7 @@
   const FTU_PING_URL = 'ftu.pingURL';
   const FTU_PING_TIMEOUT = 'ftu.pingTimeout';
   const FTU_PING_TRY_INTERVAL = 'ftu.pingTryInterval';
+  const FTU_PING_TELEMETRY_ENABLED_KEY = 'metrics.selectedMetrics.level';
```
Comment (marshall) [inline apps/system/js/ftu_ping.js]: would it make sense to use the suffix  `LEVEL` instead of `ENABLED_KEY`?

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=285e01ff839a19013610fd27f99cf419e7e71f7a

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6a3d65dad6155059bb52aab8548ba12cbe701aef

**Pull request merged**